Sources/Drain.swift

```diff
@@ -22,7 +22,7 @@ public final class Drain: DataRepresentable, Stream {
 
 
         while !stream.closed {
-            if let chunk = try? stream.receive() {
+            if let chunk = try? stream.receive(max: Int.max) {
                 buffer.bytes += chunk.bytes
```

Comment:
I'm not sure about this here it should be an arbitrary value. Unless we specify in the docs that `Int.max` means implementation default. But I think this is dangerous. It's safer to define something like 1024.